libcpuid/exports.def

```diff
@@ -30,3 +30,5 @@ cpu_clock_by_ic @26
 cpuid_get_total_cpus @27
 cpu_msr_driver_open_core @28
 cpuid_get_vendor @29
+cpu_rdmsr_range @30
+get_bits_value @31

```

Comment:
Not really. I have added `cpu_rdmsr_range` because `cpu_rdmsr` is exported.
`get_bits_value` is not useful. For the name, it can be changed. It was basically the name in i7z source code.